ui-automation-tests/support/steps/input-steps.ts

```diff
@@ -14,9 +14,12 @@ const fields = {
     "service name": "serviceName",
     "new password": "newPassword",
     "current password": "currentPassword",
-    "redirect uris": "redirectUris",
-    "post logout redirect uris": "redirectUris",
-    "change your public key": "serviceUserPublicKey"
+    "redirect uris": "redirectUri",
+    "redirect uri": "redirectUri",
```

Comment:
Do we still require both `redirect uris` and `redirect uri` for our tests?

Reply:
fixed